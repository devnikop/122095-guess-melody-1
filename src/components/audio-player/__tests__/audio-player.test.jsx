import React from "react";
import renderer from "react-test-renderer";

import AudioPlayer from "../audio-player.jsx";

const createNodeMock = (element) => {
  if (element.type === `audio`) {
    return {};
  }
};

it(`snapshot`, () => {
  const tree = renderer.create(
    <AudioPlayer
      isPlaying={false}
      onPlayButtonClick={jest.fn()}
      src={`source`}
    />,
    { createNodeMock }
  ).toJSON();

  expect(tree).toMatchSnapshot();
});
