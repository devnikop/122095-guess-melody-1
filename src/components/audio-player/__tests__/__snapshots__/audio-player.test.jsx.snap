// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
Array [
  <button
    className="track__button track__button--play"
    disabled={true}
    onClick={[MockFunction]}
    type="button"
  />,
  <div
    className="track__status"
  >
    <audio />
  </div>,
]
`;
